apps/scan/backend/test/helpers/custom_helpers.ts

```diff
@@ -123,6 +123,7 @@ export async function withApp(
     });
     mockUsbDrive.assertComplete();
   } finally {
+    await waitForContinuousExportToUsbDrive(mockUsbDrive);
```

Comment:
Because this wait will now take no time if irrelevant, we can use it more liberally. Here, I run it at the end of the `withApp` helper that we use across most VxScan backend tests